src/client/types.test.ts

```diff
@@ -30,15 +31,33 @@ describe('without the leading slash', () => {
     .get('foo', (c) => c.json({}))
     .get('foo/bar', (c) => c.json({}))
     .get('foo/:id/baz', (c) => c.json({}))
-  const client = hc<typeof app>('')
+  const client = hc<typeof app, 'http://localhost'>('http://localhost')
```

Comment:
Can we go without the second argument like this?

```ts
const client = hc<typeof routes>('http://localhost/')
```

Or impossible caused by a TypeScript limitation?

Reply:
Thank you for answering. It's a good idea. I wonder if it's the best naming, but `createHc()` might be better (but `hcStrict` also good). Anyway, first, we will not include the function in this PR.